### PR TITLE
fix: Correctly parse version numbers

### PR DESCRIPTION
**Pull request by stephenmathieson**

### CODE REVIEW

**Review by dbowling — APPROVED** [review]
Good addition.

**Review by Zidious — APPROVED** [review]
LGTM

File: update.sh [hunk 1]
```diff
@@ -46,16 +46,26 @@ echo "version=$Version" >>"$GITHUB_OUTPUT"
 CleanVersion=$(echo "$PreviousVersion" | tr -cd '[:alnum:]._-')
 echo "previous_version=$CleanVersion" >>"$GITHUB_OUTPUT"
 
-PreviousParts=("${CleanVersion//./ }")
+IFS='.' read -ra PreviousParts <<<"$CleanVersion"
```
Comment (dbowling) [inline update.sh:49]: Don't hold up for this comment, but do you need the `-r`? Are we expecting backslashes here?

Comment (stephenmathieson) [reply update.sh:49]: Good question. No, we're not expecting backslashes. Copilot wrote this for me 😅

**Pull request merged**